glossary.html

```diff
@@ -2,6 +2,7 @@
 <html>
 <head>
     <title>Shivani Sharma,s CSD120 Website</title>
+    <link rel="stylesheet" href="styles/main.css">
```

Comment:
Missing stylesheet link (it needs to be in everypage that the stylesheet applies to)